apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -32,13 +32,15 @@ function InviteToV2ProgressModal({
   setHasSeenProgressTableInvite,
   setShowProgressTableV2,
   setDateProgressTableInvitationDelayed,
+  showProgressTableV2,
 }) {
   const [invitationOpen, setInvitationOpen] = React.useState(false);
 
   React.useEffect(() => {
     if (
       dateProgressTableInvitationDelayed === undefined ||
-      hasSeenProgressTableInvite === undefined
+      hasSeenProgressTableInvite === undefined ||
+      showProgressTableV2 === undefined
```

Comment:
Due to how the user properties block serializes properties, `false` will be stripped to `nil`. Therefore this logic won't work.

Reply:
Modified to instead use the timestamp.  For this section though, we are checking for `undefined` so that if the state is not done loading it won't "glitch" and show the pop-up unintentionally.